_content/operate-connector.md

```diff
@@ -0,0 +1,377 @@
+## Hosting an Interledger node
+
+Interledger users may want to operate their own node software, rather than relying on a third party provider, just like some publishers want to host their own website.
+
+There are three primary implementations to operate an Interledger connector:
+
+- [Java Connector](https://github.com/interledger4j/ilpv4-connector) &mdash; _Actively maintained by Xpring and recommended for new production deployments._
+- [Interledger.rs (Rust)](https://github.com/interledger-rs/interledger-rs) &mdash; _Not actively developed, but recommended to get started for its ease of use and featureset._
+- [`ilp-connector` (JavaScript)](https://github.com/interledgerjs/ilp-connector) &mdash; _Battle-tested in production._
```

Comment:
Right now, there isn't a leading implementation and they each have trade-offs, which is why I chose to lay out all the options. Hopefully in the not-to-distant future this will be Java, but there aren't many resources yet (hence why I thought Rust was more suitable for this tutorial).